app/assets/javascripts/discourse/app/components/modal/revise-and-reject-post-reviewable.js

```diff
@@ -0,0 +1,52 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import { isEmpty } from "@ember/utils";
+
+const OTHER_REASON = "Other...";
```

Comment:
So this is not translated?

Reply:
Actually you bring up a good point, it is a "special" reason that always exists outside the site setting but there is no reason not to have a translated version in the dropdown. Will fix